includes/views/promo.php

```diff
@@ -19,19 +19,18 @@
 	} else {
 		$hide_ad = get_option( 'wpo_wcpdf_hide_promo_ad' );
 	}
-	
 	if ( $now->getTimestamp() < $end->getTimestamp() && ! $bundle_installed && ! $hide_ad ) :
 ?>
 
-
 <div class="wcpdf-promo-ad">
 	<img src="<?php echo esc_url( WPO_WCPDF()->plugin_url() . '/assets/images/wpo-helper.png' ); ?>" class="wpo-helper">
-	<h3><?php esc_html_e( 'Black Friday!', 'woocommerce-pdf-invoices-packing-slips' ); ?></h3>
-	<ul class="wcpdf-extensions">
-		<li>
-			<?php //TODO ?>
-		</li>
-	</ul>
-	<?php printf( '<a href="%s" style="display:inline-block; margin-top: 10px;">%s</a>', esc_url( wp_nonce_url( add_query_arg( 'wpo_wcpdf_hide_promo_ad', 'true' ), 'hide_promo_ad_nonce' ) ), esc_html__( 'Hide this message', 'woocommerce-pdf-invoices-packing-slips' ) ); ?>
+	<h3><?php esc_html_e( 'Last chance to use your Black Friday discount!', 'woocommerce-pdf-invoices-packing-slips' ); ?></h3>
+	<p>Elevate your customer experience and streamline your operations with one of our powerful extensions.</p>
+	<p>Use promocode <strong class="code">blackfriday30</strong> to get a <strong>30% discount</strong> on your upgrade!</p>
+	<?php if ( isset( $_GET['tab'] ) && $_GET['tab'] !== 'upgrade' ) : ?>
```

Comment:
When you click the "PDF Invoices" link from the menu the `tab` is not added to the URL, and in this case the message is not being displayed.